### PR TITLE
Papercuts galaxy 101

### PR DESCRIPTION
**Pull request by gallardoalba**
This PR includes the suggestion of #2398.

### CODE REVIEW

**Comment by shiltemann** [comment]
cool, thanks for doing this @gallardoalba :+1:

File: topics/introduction/tutorials/galaxy-intro-101/tutorial.md [hunk 1]
```diff
@@ -161,6 +166,8 @@ Below we describe how you can do this, but it is not necessary for this tutorial
 > >     > the gray (preparing/queued) and yellow (running) states to become green (success):
 > >     >
 > >     > ![`UCSC Main on Human: knownGene` dataset is green](../../images/101_04.png)
+> >     >
+> >     > Now you need to login once again in your account.
```
Comment (shiltemann) [inline topics/introduction/tutorials/galaxy-intro-101/tutorial.md:170]: I believe this is only true on Chrome currently, should we make it "You might need to login to Galaxy again" orso? What do you think?

Comment (gallardoalba) [reply topics/introduction/tutorials/galaxy-intro-101/tutorial.md:170]: Sure :)

**Review by shiltemann — APPROVED** [review]

**Comment by shiltemann** [comment]
thanks @gallardoalba!

**Pull request merged**